src/App.js

```diff
@@ -1,13 +1,19 @@
 import "./App.css";
 import Navbar from "./components/navbar/Navbar";
-
+import { Routes,Route } from "react-router";
 import Home from "./pages/home/Home";
+import Login from "./pages/login/Login";
+import Signup from "./pages/signup/Signup";
```

Comment:
you can create an index.js file in pages, and import all of the above from a single file

Reply:
okay